templates/account/edit.tpl

```diff
@@ -3,7 +3,7 @@
 
 <div class="account">
 	<div class="row">
-		<div class="col-md-2" style="text-align: center; margin-bottom:20px;">
```

Comment:
nice. I notice some of these inline styles don't seem to do anything in the first place haha